src/containers/landing/content.tsx

```diff
@@ -0,0 +1,16 @@
+// Welcome Modal
+export const MODAL_TITLE = {
+  ENG:
+    'Welcome to Speak for the Trees’ Adopt-A-Tree Program, run in partnership ' +
+    'with the Boston Parks and Recreation Department and Code4Community!',
+};
+export const MODAL_PARAGRAPH = {
+  ENG:
+    'Are there young, newly planted street trees near you? Explore our ' +
+    'tree map to learn how to adopt one (or a few!) and care for them to ' +
+    'help Boston’s young trees reach maturity and give your community ' +
+    'the many benefits mature trees provide.',
+};
+export const MODAL_OK_TEXT = {
+  ENG: 'Adopt a Tree!',
```

Comment:
💭 Do we want to make the "OK" text something more noncommittal so that people who just want to look at the map don't feel dissuaded? Maybe something along the lines of "Explore Boston's urban forest!"